common-functions

```diff
@@ -30,8 +30,7 @@ get_random_ports() {
     local port=$RANDOM
     local quit=0
     while [ "$quit" -ne 1 ]; do
-      netstat -an | grep $port > /dev/null
-      if [ $? -gt 0 ]; then
+      if ! netstat -an | grep $port > /dev/null; then
```

Comment:
your lint tool wanted that change ;)

see again: https://travis-ci.org/dokku/dokku-mariadb/jobs/264175402